scss/includes/_shared_app_switcher.scss

```diff
@@ -0,0 +1,108 @@
+sl-dropdown.app-switcher-dropdown,
+sl-button.app-switcher-dropdown-trigger {
+  display: block;
+}
+
+sl-button.app-switcher-dropdown-trigger::part(base) {
+  border: 1px solid var(--sl-color-neutral-400);
```

Comment:
One other CSS thing we've been doing elsewhere is to use the Tailwind `@apply` rules to set the styling. That makes it a bit more portable between the inline code and an external stylesheet. Not sure how much of this styling would directly translate to tailwind classes, though.